[OCPBUGS-24484] Add missing launchSecurity parameter and verification (#69341)
openshift/openshift-docs: modules/ibm-z-secure-execution.adoc

----
gpg --recipient-file /path/to/ignition.gpg.pub --yes --output /path/to/config.ign.gpg --verbose --armor --encrypt /path/to/config.ign
----

> . Follow the fast-track installation procedure to install nodes using the {ibm-name} Secure Execution QCOW image.

Reviewer: ```suggestion
. Follow the fast-track installation of RHCOS to install nodes by using the {ibm-name} Secure Execution QCOW image.
```

+
[NOTE]
====

> Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` and add the `launchSecurity` parameter.

Reviewer: ```suggestion
Before you start the VM, replace `serial=ignition` with `serial=ignition_crypted`, and add the `launchSecurity` parameter.
```

> Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` and add the `launchSecurity` parameter when mounting the Ignition file.

Reviewer 2: `launchSecurity` affects all IO deives and must be enabled on each VM boot. I guess it'd be better to drop `when mounting the Ignition file.`

====

.Verification

> When you have completed the fast-track installation procedure and Ignition runs on the first boot, verify if decryption is successful.

Reviewer: ```suggestion
* Verify if decryption was successful after the ignition was executed at the first boot.
```

Author: Will keep mention of fast-track installation. People tried verification before doing that.

> * If the decryption is successful, you will see an output like the following example:

Reviewer: ```suggestion
** If the decryption was successful, you can expect an output similar to the following example:
```

[    2.808874] coreos-secex-ignition-decrypt[740]: gpg: encrypted with rsa4096 key, ID <key_name>, created <yyyy-mm-dd>
[  OK  ] Finished coreos-secex-igni…S Secex Ignition Config Decryptor.
----

> * If the decryption fails, you will see an output like the following example:

Reviewer: ```suggestion
** If the decryption failed, you can expect an output similar to the following example:
```
